index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Chawki Mbarki</title>
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/c6f38472f5.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header class="toolbar">
+    <div class="shape-1"><img src="Assets/Shapes/1.svg" alt="geometry-shape"></div>
```

Comment:
- [ ] Kindly note that according to the requirement, the background image should be added with CSS [background image](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) property. 😺

Reply:
Fixed.